packages/blockchain-wallet-v4-frontend/src/components/Alerts/messages.tsx

```diff
@@ -504,8 +504,8 @@ const getAlertContent = (message, data = undefined) => {
     case C.MOBILE_LOGIN_ERROR:
       return buildMessageTemplate(
         <FormattedMessage
-          id='components.alerts.mobile_login_error'
-          defaultMessage='Error logging into your wallet'
+          id='components.alerts.mobile_login_account_error'
```

Comment:
Nice work! You'll also need to update the related strings in `blockchain-wallet-v4-frontend/packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts`

So for example, in that file you'll see an entry for `components.alerts.mobile_login_error`. Just change that text the same as you did in the template files. You'll need to do this for each of the text strings you've changed in this PR.